pytorch_lightning/plugins/__init__.py

```diff
@@ -19,21 +19,22 @@
 from pytorch_lightning.plugins.precision.sharded_native_amp import ShardedNativeMixedPrecisionPlugin
 from pytorch_lightning.plugins.precision.tpu import TPUPrecisionPlugin
 from pytorch_lightning.plugins.precision.tpu_bf16 import TPUBf16PrecisionPlugin
-from pytorch_lightning.strategies.ddp import DDPStrategy
-from pytorch_lightning.strategies.ddp2 import DDP2Strategy
-from pytorch_lightning.strategies.ddp_spawn import DDPSpawnStrategy
-from pytorch_lightning.strategies.deepspeed import DeepSpeedStrategy
-from pytorch_lightning.strategies.dp import DataParallelStrategy
-from pytorch_lightning.strategies.fully_sharded import DDPFullyShardedStrategy
-from pytorch_lightning.strategies.horovod import HorovodStrategy
-from pytorch_lightning.strategies.ipu import IPUStrategy
-from pytorch_lightning.strategies.parallel import ParallelStrategy
-from pytorch_lightning.strategies.sharded import DDPShardedStrategy
-from pytorch_lightning.strategies.sharded_spawn import DDPSpawnShardedStrategy
-from pytorch_lightning.strategies.single_device import SingleDeviceStrategy
-from pytorch_lightning.strategies.single_tpu import SingleTPUStrategy
-from pytorch_lightning.strategies.strategy import Strategy
-from pytorch_lightning.strategies.tpu_spawn import TPUSpawnStrategy
+from pytorch_lightning.plugins.training_type.ddp import DDPPlugin
```

Comment:
does the `DDPPlugin` contain a deprecation message? if not shall we try some mapping it is just about renaming classes:
```
from pytorch_lightning.strategies.ddp import DDPStrategy as DDPPlugin
```

Reply:
yes it already contains a deprecation message.